src/signals/incident/components/IncidentPreview/index.js

```diff
@@ -111,11 +118,22 @@ const heading = previewKey => {
   }
 };
 
+const getEditLinkText = (section, value) => {
+  if (section === 'beschrijf') return 'Wijzig melding';
+  if (section === 'vulaan') return 'Wijzig aanvullende informatie';
+
+  const entry = Object.entries(value)[0];
```

Comment:
This construction makes too many assumptions about the data structure. You could create the text directly at the source (`src/signals/incident/definitions/wizard-step-5-samenvatting.js`) in the previewFactory.

Reply:
Moved it to samenvatting step definition